### ISSUE TITLE
[Bug]: Chromium 114 might bring an accessibility issue with JAWS 2021 and older

### ISSUE THREAD

**Issue by benbeaudry** [electron/electron#39076]
### Preflight Checklist

- [X] I have read the [Contributing Guidelines](https://github.com/electron/electron/blob/main/CONTRIBUTING.md) for this project.
- [X] I agree to follow the [Code of Conduct](https://github.com/electron/electron/blob/main/CODE_OF_CONDUCT.md) that this project adheres to.
- [X] I have searched the [issue tracker](https://www.github.com/electron/electron/issues) for a bug report that matches the one I want to file, without success.

### Electron Version

25.0.0

### What operating system are you using?

Windows

### Operating System Version

Windows 11

### What arch are you using?

x64

### Last Known Working Electron version

24.0.0

### Expected Behavior

JAWS 2021 and older should be able to read the web content of an Electron app. 

### Actual Behavior

There's a chance (we have seen this issue in Chrome, Edge and WebView2, but couldn't test with Electron 25 yet) that JAWS 2021 does not work in the web content anymore. For example, tabbing to a specific element in the web content of an Electron app would not be announced by the screen reader.

This is just a heads-up. If you do encounter this error, please reach out to me and the Google Chrome accessibility team.

**Comment by VerteDinde** [comment]
@benbeaudry Thanks so much for the heads up, it's very appreciated! Sorry, I'm not as familiar with JAWS - do you happen to have a link to the upstream Chrome bug, and we can track it? I'll try a repro in Electron this week and we can confirm if we're experiencing it as well 🙇‍♀️

**Comment by benbeaudry** [comment]
I believe the Chromium bug was https://bugs.chromium.org/p/chromium/issues/detail?id=1450993&q=jaws%202021&can=1.

However, you will have the fix to this bug in 114. There was a separate issue in WebView2 and there are no publicly available bugs I can share about this one, and this GitHub issue is about that WebView2 issue we faced & fixed.
